app/submitter/converter.py

```diff
@@ -88,7 +88,13 @@ def prepare_answers(metadata_store, questionnaire, answers):
             if item is not None:
                 value = answers[key]
                 if value is not None:
-                    data[item.code] = Converter._encode_value(value)
+                    if item.code not in data:
```

Comment:
This is fine for now, we need to refactor the submitter to use the answer store, but that can be done in another pull request.